how-to/openldap/install-openldap.md

```diff
@@ -1,4 +1,11 @@
 (install-openldap)=
+
+Lightweight Directory Access Protocol (LDAP) is a protocol used for managing hierarchical data. It offers a way to store, organise and manage an organisation's data such as employee accounts and computers. It facilitates centralised authentication and authorisation management.
```

Comment:
It would be really good to include a link over "Lightweight Directory Access Protocol" to the official LDAP homepage or documentation for the benefit of readers who might want to know more.